doc/examples/mzml-mzqc-example.mzML

```diff
@@ -341,7 +340,7 @@
 				<cvParam cvRef="MS" accession="MS:1000285" name="total ion current" value="6.937649e06"/>
 				<cvParam cvRef="MS" accession="MS:1000528" name="lowest observed m/z" value="300.000828877017"/>
 				<cvParam cvRef="MS" accession="MS:1000527" name="highest observed m/z" value="2008.458458829989922"/>
-				<cvParam cvRef="QC" accession="QC:4000268" name="Min. peaks for 50% of total spectra intensity" value="11"/>
+				<cvParam cvRef="MS" accession="MS:4000068" name="spectra half-TIC" value="{'MS:1000767': [1011], 'UO:0000191': [0.0235]}"/>
```

Comment:
This is not a correct value for the nativeID. Probably scan number instead again?